runtime/mux.go

```diff
@@ -154,6 +163,17 @@ func WithOutgoingHeaderMatcher(fn HeaderMatcherFunc) ServeMuxOption {
 	}
 }
 
+// WithOutgoingTrailerMatcher returns a ServeMuxOption representing a headerMatcher for outgoing response from gateway.
+//
+// This matcher will be called with each header in response trailer metadata. If matcher returns true, that header will be
+// passed to http response returned from gateway. To transform the header before passing to response,
+// matcher should return modified header.
```

Comment:
I know this is the same language as for `WithOutgoingHeaderMatcher`, but do you mind correcting it too?
```suggestion
// matcher should return the modified header.
```